internal/stacks/stackruntime/internal/stackeval/stubs/unknown.go

```diff
@@ -179,9 +183,16 @@ func (u *unknownProvider) ImportResourceState(request providers.ImportResourceSt
 }
 
 func (u *unknownProvider) MoveResourceState(request providers.MoveResourceStateRequest) providers.MoveResourceStateResponse {
-	// This is offline functionality, so we can hand it off to the unconfigured
-	// client.
-	return u.unconfiguredClient.MoveResourceState(request)
+	var diags tfdiags.Diagnostics
+	diags = diags.Append(tfdiags.AttributeValue(
+		tfdiags.Error,
+		"Called MoveResourceState on a configured provider",
```

Comment:
wouldn't this be an unconfigured provider?

Reply:
Yep, this was a copy-paste from elsewhere. I've updated so it says "unknown provider" explicitely.